src/components/SingleTrikotActual.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react'
+import {List, Image,Container} from "semantic-ui-react";
+import { connect } from 'react-redux'
+import store from "../store";
+import * as riderStageConnectionActions from "../actions/riderStageConnectionsActions";
+
+class SingleTrikotActual extends Component {
+
+    render() {
+        const trikot = this.props.data;
+        const {actualStage} = this.props;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "actualStage" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of the unused 'actualStage' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AUnusedVariable)